client/src/api/users.js

```diff
@@ -0,0 +1,25 @@
+const API_URL = "http://localhost:4000";
+
+const headers = {
+  Accept: "application/json",
+  "Content-Type": "application/json",
+  "Access-Control-Allow-Origin": window.location.origin
+};
+
+export const userSignUp = async (email, password) => {
```

Comment:
Don't forget to remove sign up in your final version, we don't want everyone to be able to register

Reply:
I've commented everything out right now, and the major clean up for the entire app is coming up soon :)